edgelet/aziot-edged/src/main.rs

```diff
@@ -70,6 +72,10 @@ async fn run() -> Result<(), EdgedError> {
         )
     })?;
 
+    let migc_filename = settings.homedir().join("migc");
+    /*log::info!("creating file at: {}", migc_filename.to_str().unwrap());
+    let _file = File::create(migc_filename.clone()).unwrap();*/
```

Comment:
Recommend adding TODO to:
1. Move migc_filename down to where it is used
2. Remove comment